src/components/buttton/index.jsx

```diff
@@ -1,5 +1,5 @@
 
-export const Button = ({...props}) => {
+export const Button = ({diplaytext, ...props}) => {
     return (
         <button {...props} style={{
```

Comment:
I recommend use styled-components or use the boostrap package you have installed to style the components. If you use the native tag `style` maybe you will get some trouble to work with @media queries later (responsiveness)